spec/features/routing/customers_auths/batch_update_spec.rb

```diff
@@ -0,0 +1,307 @@
+          check :Src_number_min_length
+          check :Src_number_max_length
+          fill_in :src_number_min_length, with: changes[:src_number_min_length]
+          fill_in :src_number_max_length, with: changes[:src_number_max_length]
+          expect do
+            subject
+            expect(page).to have_selector '.flash', text: success_message
+          end.to have_enqueued_job(AsyncBatchUpdateJob).on_queue('batch_actions').with 'CustomersAuth', be_present, changes, be_present
+        end
+      end
+    end
+
+    context '"dst_number_max_length"' do
+      context 'should have error:' do
+        it "can't be blank and is not a number" do
+          check :Dst_number_max_length
+          fill_in :dst_number_max_length, with: nil
+          click_button :OK
+          expect(page).to have_selector '.flash', text: "can't be blank"
+          expect(page).to have_selector '.flash', text: 'is not a number'
+        end
+
+        it 'must be changed together' do
+          check :Dst_number_max_length
+          fill_in :dst_number_max_length, with: 12
+          click_button :OK
+          expect(page).to have_selector '.flash', text: 'must be changed together'
+        end
+      end
+
+      context 'should change min and max together' do
+        let(:changes) { { dst_number_min_length: '10', dst_number_max_length: '20' } }
+        it 'must be changed together' do
+          check :Dst_number_min_length
+          check :Dst_number_max_length
+          fill_in :dst_number_min_length, with: changes[:dst_number_min_length]
+          fill_in :dst_number_max_length, with: changes[:dst_number_max_length]
+          expect do
+            subject
+            expect(page).to have_selector '.flash', text: success_message
+          end.to have_enqueued_job(AsyncBatchUpdateJob).on_queue('batch_actions').with 'CustomersAuth', be_present, changes, be_present
+        end
+      end
+    end
+
+    context '"dst_number_min_length" should have error:' do
+      it "can't be blank and is not a number" do
+        check :Dst_number_min_length
+        fill_in :dst_number_min_length, with: nil
+        click_button :OK
+        expect(page).to have_selector '.flash', text: "can't be blank"
+        expect(page).to have_selector '.flash', text: 'is not a number'
+      end
+
+      it 'must be changed together' do
+        check :Dst_number_min_length
+        fill_in :dst_number_min_length, with: 12
+        click_button :OK
+        expect(page).to have_selector '.flash', text: 'must be changed together'
+      end
+    end
+
+    context '"dst_number_max_length" should have error:' do
+      before { check :Dst_number_max_length }
+      it "can't be blank and is not a number" do
+        fill_in :dst_number_max_length, with: nil
+        click_button :OK
+        expect(page).to have_selector '.flash', text: "can't be blank"
+        expect(page).to have_selector '.flash', text: 'is not a number'
+      end
+
```

Comment:
it is to expensive to run feature test for each attribute validator, also tests has not clear structure 

suggestion 
```
subject do
  fill_form
  click_button :OK
end 

context 'when form is invalid' do 
   let(:fill_form) do 
         
   end
end

context 'when form is valid' do
   let(:fill_form) do 
         
   end
end
end
```